app/redux/selectors/ConnectedDeviceStatusSelectors.js

```diff
@@ -0,0 +1,7 @@
+const stateRoot = (state) => state.connectedDevice;
+
+export const getConnectedDeviceStatus = (state) => stateRoot(state).status;
+
+export const getConnectedDeviceName = (state) => stateRoot(state).deviceName;
+
+export const getConnectedDeviceIdentifier = (state) => stateRoot(state).deviceIdentifier;
```

Comment:
new line

Reply:
Done and also refactored placement of connected device folder/component